lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_model_base.rb

```diff
@@ -246,21 +246,31 @@ def self.normalize(str)
       arr.join
     end
 
-    def method_missing(m, *args)
+    def method_missing(method_name, *args)
       #
       # Normalize result of any method call
       #  e.g. normalized_ldap_group, will call ldap_group method and normalize the result
       #
       prefix = 'normalized_'
-      if m.to_s.starts_with?(prefix)
-        method = m.to_s[prefix.length..-1]
+      if method_name.to_s.starts_with?(prefix)
+        method = method_name.to_s[prefix.length..-1]
         result = MiqAeServiceModelBase.wrap_results(object_send(method, *args))
         return MiqAeServiceModelBase.normalize(result)
       end
 
       super
     end
 
+    def respond_to_missing?(method_name, include_private = false)
+      prefix = 'normalized_'
```

Comment:
I like the inclusion of `respond_to_missing?` but with the duplication of this string I think it would be best to make it a constant now.

Reply:
Good point 👍